.github/workflows/main.yml

```diff
@@ -8,10 +8,12 @@ jobs:
       - name: Checkout Repository
         uses: actions/checkout@master
       - name: Install Package Dependencies
+        env:
+          GH_PAT: ${{ secrets.GH_PAT }}
         run: |-
           Rscript -e "install.packages('remotes', repos = 'https://demo.rstudiopm.com/all/__linux__/bionic/latest')"
-          Rscript -e "remotes::install_deps(dependencies = TRUE, repos = 'https://demo.rstudiopm.com/all/__linux__/bionic/latest')"
-          Rscript -e "install.packages(c('testthat','RCurl','questionr','choroplethr'))"
+          Rscript -e "install.packages(c('testthat','RCurl','questionr'))"
+          Rscript -e "devtools::install_github('p8105/p8105.datasets',auth_token=Sys.getenv('GH_PAT'))"
```

Comment:
@madalyn-benson I figured it out!  Please add the lines in green above and delete the lines in red from your .github/workflows/main.yml file.  Then push again.